node/src/Transaction.ts

```diff
@@ -2097,6 +2098,35 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createZMPop(keys, modifier, count));
     }
 
+    /**
+     * Pops a member-score pair from the first non-empty sorted set, with the given `keys` being
```

Comment:
Same doc suggestions as in BaseClient.ts. Can you also include the "blocking commands" wiki link here?